bin/publisher/robot.yaml

```diff
@@ -2,11 +2,11 @@
 # https://github.com/robocorp/rcc/blob/master/docs/recipes.md#what-is-in-robotyaml
 
 tasks:
-  Generate the whole gallery:
-    shell: python -m robocorp.tasks run init-gallery.py
+  Build all packages:
+    shell: python -m robocorp.tasks run build-all-packages.py
 
-  Update gallery with a single pack:
-    shell: python -m robocorp.tasks run update-gallery.py
+  Build updated packages:
+    shell: python -m robocorp.tasks run build-updated-packages.py
```

Comment:
If we build all packages previously, do we need to build the updated ones again? 🤔

Reply:
I'm not quite sure I'm following... Building all packages is a separate workflow, which will only happen if there is no manifest (and therefore no gallery) present in the S3. Ideally, it will only happen once, but still wanted to keep it as a separate workflow, so people don't get confused (and we can actually safeguard from accidental "reset" by adding some additional guard step in the init workflow).

Otherwise, update flow will only build packages that actually changes (the version is different).